src/guide/routing.md

```diff
@@ -1,12 +1,12 @@
-# Routing
+# Roteamento
 
-## Official Router
+## Roteador oficial (Official Router)
 
-For most Single Page Applications, it's recommended to use the officially-supported [vue-router library](https://github.com/vuejs/vue-router). For more details, see vue-router's [documentation](https://router.vuejs.org/).
+Para a maioria das aplicações de página única (SPA), é recomendado utilizar a [biblioteca vue-router](https://github.com/vuejs/vue-router) oficial. Para mais detalhes, veja a [documentação](https://router.vuejs.org/) do `vue-router`.
 
-## Simple Routing from Scratch
+## Roteamento básico do zero
 
-If you only need very simple routing and do not wish to involve a full-featured router library, you can do so by dynamically rendering a page-level component like this:
+Se você só precisa de um roteamento simples e não tem interesse de utilizar uma biblioteca robusta de roteamento, você pode o fazer dinamicamente, renderizando um componente no nível da página (page-level component), desse jeito: 
```

Comment:
Por gentileza, corrigir o "de" em _"e não tem interesse de utilizar"_ por "em", já que expressamos interesse _em_ algo.
Portanto: "e não _possui_ interesse em utilizar"

Também utilizamos, na documentação, os termos originais - quando se fazem necessário - com itálico. Portanto:

```
renderizando um componente no nível da página (page-level component)
```

se torna:

```
renderizando um componente no nível da página _(page-level component)_
```

Também sugiro a atualização de "desse jeito" por "deste modo".